rest/src/main/webapp/js/audit.js

```diff
@@ -35,7 +35,187 @@ angular
                 $scope.repoName = $stateParams.name;
                 $scope.account = $stateParams.owner;
             }])
+    .directive('propertyAudit', ['store', function(store)
+    {
+        return {
+            restrict: 'A',
+            templateUrl: 'repo/audit/propertyAudit.tpl.html',
+            scope: true,
+            controller:
+                ['$scope', '$http', '$stateParams', '$httpParamSerializer', '$filter', 'store', 'contextService',
+                    function($scope, $http, $stateParams, $httpParamSerializer, $filter, store, contextService)
+                    {
+                        var t = store.get("repoAuditTypes"),
```

Comment:
Is there any portion of this that can be merged from another function?  I feel like some of this common input patterns can be re-used from the regular history view.  I may be wrong though.